wolftpm/tpm2_packet.h

```diff
@@ -26,16 +26,16 @@
 
 #define TPM2_HEADER_SIZE 10 /* expected TPM2 header size */
```

Comment:
Should this be set to `sizeof(TPM2_HEADER)` just in case it ever gets used?

Reply:
No, because it would not be guaranteed to be 10 due to not being packed. I thought the note below about it not being used was good enough, but it can be totally removed. I like it there for reference purposes.

Reply:
Makes sense.